extensions/windows-event-log/ConsumeWindowsEventLog.h

```diff
@@ -101,6 +101,7 @@ class ConsumeWindowsEventLog : public core::Processor
   
 
 protected:
+  void refreshTimeZoneData();
```

Comment:
why not private?

Reply:
All the internally used functions of the class are protected currently, didn't really want to diverge as it doesn't hurt.